### PR TITLE
Update adventures.json

### PR DESCRIPTION
**Pull request by LEDFlashing**

### CODE REVIEW

File: app/config/tstvhq/adventures.json [hunk 1]
```diff
@@ -3,19 +3,19 @@
         {
             "config": {
                 "audio": {
-                    "path": "https://jitsi-party.s3.amazonaws.com/Satanic+Planet+999+(official+music+video).mp3",
+                    "path": "https://drive.google.com/file/d/142NtV56pgDTcQuBpHSUef6uJVDiqiStp/view?usp=sharing",
```
Comment (hndrewaall) [inline app/config/tstvhq/adventures.json:6]: Needs to be S3 URL. Send me a link to a file i can download myself and upload to S3. Can't figure out how to use that URL

File: app/config/tstvhq/adventures.json [hunk 2]
```diff
@@ -26,60 +26,60 @@
...
                           "nextRoom": ["library"]
                       }
                     ]
                 },
                 {
                     "id": "card",
-                    "name": "The Card",
-                    "text": "For possibly the first time, divination has correctly told you something true.  The card reads, ”You are going to die someday.“",
+                    "name": "listen",
+                    "text": "You listen.“",
```
Comment (hndrewaall) [inline app/config/tstvhq/adventures.json:60]: not technically a syntax error since directional quotes don't count, but it's damn confusing

**Review by hndrewaall — CHANGES_REQUESTED** [review]
2 changes

**Comment by kacycrider** [comment]
Hey Jack! Could you please swap the drive audio file for this link? Greatly appreciate you! 

https://tstvhq.s3.amazonaws.com/music/elevator_ritual.mp3

File: app/config/tstvhq/adventures.json (hunk 1 above)
Comment (LEDFlashing) [reply app/config/tstvhq/adventures.json]: changed

File: app/config/tstvhq/adventures.json (hunk 2 above)
Comment (LEDFlashing) [reply app/config/tstvhq/adventures.json]: fixed

**Review by hndrewaall — APPROVED** [review]

**Pull request merged**